.github/workflows/unit-tests.yml

```diff
@@ -60,10 +63,10 @@ jobs:
         run: vendor/bin/phpunit --configuration phpunit.xml --coverage-text --exclude-group dns
 
       - name: MySQL Testing
-        run: vendor/bin/phpunit --configuration phpunit-mysql-workflow.xml --exclude-group dns
+        run: vendor/bin/phpunit --configuration phpunit-mysql.xml --exclude-group dns || echo 'debug'
```

Comment:
@DarkSide666 Upps, please fix (remove ` || echo 'debug'`) this immediatelly, I can not do it easily :(

Reply:
Done.